content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -0,0 +1,161 @@
+
+In the 1.4 release, the Istio controller is still in the alpha phase of development and not fully
+integrated with `istioctl`. It is, however, available for experimentation using `kubectl` commands.
+For example, to install the controller and a default version of Istio into your cluster,
+run the following command:
+
+{{< text bash >}}
+$ kubectl apply -f https://<repo URL>/<version>/operator-profile-default.yaml
+{{< /text >}}
+
+You can then make changes to the Istio installation configuration:
+
+{{< text bash >}}
+$ kubectl edit istiocontrolplane example-istiocontrolplane -n istio-system
+{{< /text >}}
+
+To upgrade to a new version of Istio, run:
+
+{{< text bash >}}
+$ kubectl apply -f https://<repo URL>/<new version>/operator-profile-default.yaml
+{{< /text >}}
+
+Refer to [standalone operator](/docs/ops/setup/standalone-operator/) for more details.
+
+## Using the {{< istioctl >}} CLI
+
+In the meantime, until the controller is thouroughly tested and better integrated with `istioctl`,
+the preferred way to use the operator API is through a new set of `istioctl` CLI commands.
+The same code is executed, the main difference being the execution context.
+In the CLI case, the operation runs in the admin user’s CLI execution and
+security context, while in the controller case, a pod in the cluster runs the code in its security context.
+In both cases, the API is schematized and validated.
+
+To install Istio into a cluster:
+
+{{< text bash >}}
+$ istioctl manifest apply -f <your-config-crd>
+{{< /text >}}
+
+Make changes to the installation configuration by editing the configuration
+file and calling `istioctl manifest apply` again.
+
+To upgrade to a new version of Istio:
+
+{{< text bash >}}
+$ istioctl x upgrade -f <your-config-changes-crd>
+{{< /text >}}
+
+In additon to specifying the complete configuration in an `IstioControlPlane` CRD,
+the `istioctl` commands can also be passed individual settings using a `--set` flag.
+There are also a number of other `istioctl` commands that, for example,
+help you list, display, and compare configuration profiles and manifests.
+Refer to the Istio [install instructions](/docs/setup/install/) for more details.
+
+## Migration from Helm
+
+The `istioctl` command includes several features to help ease the trasition from previous
+configurations using Helm:
+
+1. Pass Helm configuration options using `istioctl --set` by prepending the string “values.“
+    to the option name. For example, instead of this helm command:
+
+    {{< text bash >}}
+    $ helm template ... --set global.mtls.enabled=true
+    {{< /text >}}
+
+    You can use this `istioctl` command:
+
+    {{< text bash >}}
+    $ istioctl manifest generate ... --set values.global.mtls.enabled=true
+    {{< /text >}}
+
+1. Automated migration from helm `values.yaml` using the `istioctl manifest migrate` command (alpha in Istio 1.4).
+
+1. The full helm installation API can also be accessed as a pass-through from `istioctl` and controller.
```

Comment:
This point is essentially saying the same thing as the first point. You could combine them for example like this:
```
The full helm installation API can also be accessed as a pass-through from `istioctl` and controller.  Pass Helm configuration options using `istioctl --set` by prepending the string “values.“  to the option name. For example, instead of this helm command:
---
example
---
 See [Customize Istio settings using the Helm API](/docs/setup/install/operator/#customize-istio-settings-using-the-helm-api)
￼   for details. 
```